cmd/podman/artifact/list.go

```diff
@@ -95,10 +97,18 @@ func outputTemplate(cmd *cobra.Command, lrs []*entities.ArtifactListReport) erro
 		if err != nil {
 			return err
 		}
-		// TODO when we default to shorter ids, i would foresee a switch
-		// like images that will show the full ids.
+
+		artifactHash := artifactDigest.Encoded()
+		// If the user does not want truncated hashes
+		formatHash := func() string {
```

Comment:
Not sure why we would need to define a extra function here.
```
if !listFlag.noTrunc {
	artifactHash = artifactHash[0:12]
}
```

This would do the same thing without an extra function call.

Reply:
i meant to delete line 101.

Reply:
reverted ... i felt funny about the fact that having a conditional check on something that was by default ... but i inverted the logic a tad, see how you like it.